packages/features/tasker/tasks/triggerNoShow/schema.ts

```diff
@@ -0,0 +1,57 @@
+import { z } from "zod";
+
+import { TIME_UNIT } from "@calcom/features/ee/workflows/lib/constants";
+import { WebhookTriggerEvents } from "@calcom/prisma/enums";
+
+const commonSchema = z.object({
+  triggerEvent: z.enum([
+    WebhookTriggerEvents.AFTER_HOSTS_CAL_VIDEO_NO_SHOW,
+    WebhookTriggerEvents.AFTER_GUESTS_CAL_VIDEO_NO_SHOW,
+  ]),
+  bookingId: z.number(),
+});
+
+export const ZWebhook = z.object({
+  id: z.string(),
+  subscriberUrl: z.string().url(),
+  appId: z.string().nullable(),
+  secret: z.string().optional(),
+  time: z.number(),
+  timeUnit: z.enum(TIME_UNIT),
+  eventTriggers: z.array(z.string()),
+  payloadTemplate: z.string().nullable(),
+});
+
+export type TWebhook = z.infer<typeof ZWebhook>;
+
+export const triggerNoShowPayloadSchema = z.object({
+  total_count: z.number(),
+  data: z.array(
+    z
+      .object({
+        id: z.string(),
+        room: z.string(),
+        start_time: z.string(),
```

Comment:
I am getting this zod error when I trigger the corn job and host joined cal video: 

```
ZodError: [
  {
   "code": "invalid_type",
    "expected": "string",
    "received": "number",  
    "path": [
         "data",
        0,
        "start_time"
     ],
     "message": "Expected string, received number"

```